src/redux/storeState.ts

```diff
@@ -0,0 +1,37 @@
+import { Moment } from "moment";
+import { Set } from "../types/LiftTypes";
+import { InputMode, LiftInfoLink } from "../types/LiftTypes";
+import { LiftLogState } from "./liftLogState";
+
+export type NewEntryState = {
+  date: Moment | null;
+  name: string;
+  weightLiftedString: string;
+  weightLifted: number | null;
+};
+
+export type DialogState = {
+  isOpen: boolean;
+
+  inputMode: InputMode;
+
+  numberOfSetsString: string;
+  numberOfRepsString: string;
+
+  numberOfSets: number;
+  numberOfReps: number;
+
+  customSetsStrings: ReadonlyArray<string>;
+  customSets: ReadonlyArray<Set>;
+
+  commentIsShown: boolean;
+  comment: string;
+  canAddLink: boolean;
+  links: ReadonlyArray<LiftInfoLink>;
+};
+
+export type StoreState = {
```

Comment:
hm... I would call it `AppState` since `store` is a different concept in the `redux` world, check [this](https://redux.js.org/api/store) on redux store on the official documentation.